Python/peephole.c

```diff
@@ -353,11 +365,16 @@ PyCode_Optimize(PyObject *code, PyObject* consts, PyObject *names,
                            stack effect */
                         h = set_arg(codestr, i, get_arg(codestr, tgt));
                     } else {
+                        Py_ssize_t arg = (tgt + 1);
```

Comment:
Maybe make `arg` and `tgt` an `unsigned int`?

Reply:
tgt is initialized by "tgt = find_op(codestr, codelen, h);" and find_op() return type is Py_ssize_t. It seems simpler to me to use Py_ssize_t here as well.